sdk/scheduler/src/test/java/com/mesosphere/sdk/offer/evaluate/security/CertificateNamesGeneratorTest.java

```diff
@@ -83,8 +98,12 @@ public void testGetSANs() throws Exception {
         Assert.assertTrue(names.contains(taskDnsName(TestConstants.TASK_NAME, TestConstants.SERVICE_NAME)));
         Assert.assertFalse(names.contains(taskDnsName("*", TestConstants.SERVICE_NAME)));
         Assert.assertFalse(names.contains(taskVipName("*", TestConstants.SERVICE_NAME)));
-        // echo -n "some-pod-test-task-name.service-name.autoip.dcos.thisdcos.directory" | sha1sum
-        Assert.assertEquals("a22fd2735aae7c55e47bece5f6c10612866583bf", certificateNamesGenerator.getSANsHash());
+
+        // Expected sha1sum of the hostname:
+        byte[] digest = SHA1_HASHER.digest("some-pod-test-task-name.service-name.autoip.tld".getBytes(StandardCharsets.UTF_8));
```

Comment:
This is repeated here quite a bit. Consider breaking out into a helper.

Reply:
Done